src/workers/parser/getScopes/visitor.js

```diff
@@ -540,7 +540,10 @@ const scopeCollectionVisitor = {
           state.sourceId
         );
       });
-    } else if (t.isImportDeclaration(node)) {
+    } else if (
+      t.isImportDeclaration(node) &&
+      (!node.importKind || node.importKind === "value")
+    ) {
```

Comment:
do we also want a unit test for this too? 

it should be simpler and would give us some quicker feedback than the mochitest, which is also helpful but less direct